src/utils/date.ts

```diff
@@ -13,3 +13,10 @@ export const convertToLocalTime = (date: Date): Date => {
 
   return isoDate;
 };
+
+export type NullableDate = Date | null | undefined;
+
+export const daysBetween = (fromDate: NullableDate, toDate?: NullableDate): number => {
```

Comment:
maybe an idea to include a small comment explaining what it does, because I am not sure what it does.